tests/test_recorders.py

```diff
@@ -1677,6 +1677,26 @@ def test_annual_total_flow_recorder_factored(self, simple_linear_model):
         df = rec_fact.to_dataframe()
         assert_allclose([[3650.0*3]], df.values)
 
+    @pytest.mark.parametrize("end_date, expected", [("2013-01-04", [30.0, 40.0]), ("2012-12-31", [30.0]),])
+    def test_annual_total_flow_recorder_year_end(self, simple_linear_model, end_date, expected):
+        """
+        Test AnnualTotalFlowRecorder when timestep crosses year end
+        """
```

Comment:
Can we add some longer explanation of the parametrized data here? I think I understand it, but would be best if it were written down that these two tests are each running 1 time-step but the first one overlaps into 2013.

Reply:
done